lms/djangoapps/edraak_forus/models.py

```diff
@@ -0,0 +1,55 @@
+"""
+Edraak ForUs models
+"""
+import logging
+from django.db import models
+from django.contrib.auth.models import User
+from django.utils.translation import ugettext_lazy as _
+from django.utils import timezone
+
+log = logging.getLogger(__name__)
+
+
+class ForusCreateUserProfile(Exception):
+    """
+    Cannot create profile for superusers and staff
+    """
+    pass
+
+
+class ForusProfile(models.Model):
+    """
+    ForUs users model
+    """
+    user = models.ForeignKey(User, unique=True, db_index=True)
```

Comment:
I know it's my fault! but this should be a `OneToOneField` according to Django:

```
System check identified some issues:

WARNINGS:
edraak_forus.ForusProfile.user: (fields.W342) Setting unique=True on a ForeignKey has the same effect as using a OneToOneField.
	HINT: ForeignKey(unique=True) is usually better served by a OneToOneField.
```

Please delete the migrations before making a new one.